var/spack/repos/builtin/packages/blitz/package.py

```diff
@@ -17,6 +17,15 @@ class Blitz(AutotoolsPackage):
     version("1.0.2", sha256="500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2")
 
     depends_on("python@3:", type="build")
+    depends_on("m4", type="build")
+    depends_on("autoconf", type="build")
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
+
+    patch("fujitsu_compiler_specfic_header.patch", when="%fj")
+
+    def autoreconf(self, spec, prefix):
+        autoreconf("--install", "--verbose", "--force")
```

Comment:
This should be the default. Why was it necessary to add this function?

Reply:
@alalazo 
Fixed. This function currently is removed now.